src/main/java/uk/ac/ebi/embl/api/validation/check/file/FileValidationCheck.java

```diff
@@ -236,8 +237,18 @@ public void validateSequencelessChromosomes() throws ValidationEngineException
 		throw new ValidationEngineException("Sequenceless chromosomes are not allowed in assembly : "+String.join(",",sequencelessChromosomes),ReportErrorType.VALIDATION_ERROR);
 
 	}
-	
-	
+
+	public void verifyUnlocalisedObjectNames() throws ValidationEngineException {
+		if( sharedInfo.unlocalisedEntryNames.size() > 0) {
+			return;
+		}
+		for (String unlocalisedEntryName : sharedInfo.unlocalisedEntryNames) {
+			if (!sharedInfo.sequenceInfo.containsKey(unlocalisedEntryName.toUpperCase())) {
+				throw new ValidationEngineException("Unlocalised Object name is not available in any of the file(flatfile/fasta/agp) : " + unlocalisedEntryName,
```

Comment:
"Unlocalised list file contains a sequence name that is not found in this submission " + unlocalisedEntryName